test/BaseCommand.test.js

```diff
@@ -1,104 +1,126 @@
 const { stdout } = require('stdout-stderr')
 const { Command } = require('@oclif/core')
+const TheCommand = require('../src/BaseCommand')
 
 jest.mock('@adobe/aio-lib-core-config')
 const mockAioConfig = require('@adobe/aio-lib-core-config')
 
 const mockConfigLoader = require('@adobe/aio-cli-lib-app-config')
 jest.mock('@adobe/aio-cli-lib-app-config')
-// const getMockConfig = require('./data-mocks/config-loader')
 
 const libEnv = require('@adobe/aio-lib-env')
 jest.mock('@adobe/aio-lib-env')
 
-const TheCommand = require('../src/BaseCommand')
-
 jest.mock('inquirer')
 const inquirer = require('inquirer')
 const mockExtensionPrompt = jest.fn()
 inquirer.createPromptModule = jest.fn().mockReturnValue(mockExtensionPrompt)
 
+let command
+
 beforeEach(() => {
+  command = new TheCommand()
+  command.argv = []
```

Comment:
I thought args was now an {} ... probably doesn't make a difference.

Reply:
oops, didn't check this comment before merging. Will fix

Reply:
actually, looking at it more, no argv is correct (essentially like process.argv). args is a separate thing for the command definition